versions.sh

```diff
@@ -95,6 +96,13 @@ for version in "${versions[@]}"; do
 		if [ "$rcVersion" = '8.0' ] && [[ "$suite" = alpine* ]] && [ "$suite" != 'alpine3.16' ]; then
 			continue
 		fi
+		# 8.0 doesn't support OpenSSL 3, which is the only version in bookworm
+		# only keep two variants of Debian per version of php
+		if [ "$rcVersion" = '8.0' ] && [ "$suite" = 'bookworm' ]; then
```

Comment:
Normally I would complain that we're hard-coding `bookworm` here in a way that means we'll have to update it again for `trixie` but PHP 8.0 should be EOL long before then (later this year!!) :smile: